### ISSUE TITLE
Front page doesn't display same events as event archive

### ISSUE THREAD

**Issue by baxil** [RogerDodger/WriteOff#33]
I noticed the "No Turning Back" event was missing from the site front as I was trying to double-check that I'd created FIMFiction folders for everything, and I'm not sure why it's gone — I thought it displayed the chronologically most recent events?

The ones currently displayed on the site front:
The Other Side · FiM Short Story · 18 May 2018
Message in a Bottle · Original Short Story · 27 Apr 2018
Just Like Old Times · FiM Minific · 7 Apr 2018
Last Call · Original Minific · 17 Mar 2018
The Next Generation · Original Polished Story · 13 Feb 2018
Long Story Short · Original Minific · 23 Dec 2017

All recent events, from the archives page:
The Other Side · FiM Short Story · 18 May 2018
Message in a Bottle · Original Short Story · 27 Apr 2018
Just Like Old Times · FiM Minific · 7 Apr 2018
Last Call · Original Minific · 17 Mar 2018
No Turning Back · FiM Short Story · 23 Feb 2018
The Next Generation · Original Polished Story · 13 Feb 2018
Under the Surface · Original Short Story · 2 Feb 2018
Lie Me a River · FiM Minific · 13 Jan 2018
Long Story Short · Original Minific · 23 Dec 2017

**Comment by RogerDodger** [comment]
Events are considered active if it or its most recent post are less than 2 months old.

This behaviour allows old events to eventually move off the front page, while also allowing people to revive an old event and have their comment seen.

I think your confusion is that the active events are sorted by creation date, but the selection criteria is mostly determined by last post date. Typically, selection and sorting criteria are the same (comments bump a thread to the top), so it's somewhat confusing.

It would be clearer that this is the behaviour if active events were sorted by last post, but I prefer it this way since more recent events are more relevant.
